apps/src/templates/teacherDashboard/OwnedSections.jsx

```diff
@@ -70,13 +60,12 @@ class OwnedSections extends React.Component {
   }
 
   addSection = () => {
-    const { defaultCourseId, defaultScriptId } = this.props;
     if (experiments.isEnabled(SECTION_FLOW_2017)) {
-      this.props.beginEditingNewSection(defaultCourseId, defaultScriptId);
+      this.props.beginEditingNewSection();
     } else {
       // This is the only usage of the newSection action, and can be removed once
       // SECTION_FLOW_2017 is finished
-      return this.props.newSection(defaultCourseId);
+      return this.props.newSection();
```

Comment:
Not blocking this PR (this is really my bad) but note to self: We should have a unit test for OwnedSections.jsx that clicks this button both with the experiment on and off.